internal/metrics/embeddings_metrics.go

```diff
@@ -0,0 +1,108 @@
+// Copyright Envoy AI Gateway Authors
```

Comment:
Looks like you ignored my previous comment, but the entire code here should not be copied and pasted but instead you could reuse almost all the code from the existing chat_completion_metrics.go by passing "operation". 

https://opentelemetry.io/docs/specs/semconv/registry/attributes/gen-ai/#gen-ai-operation-name